include_core/omrport.h

```diff
@@ -2209,6 +2209,8 @@ typedef struct OMRPortLibrary {
 	uintptr_t (*str_ftime)(struct OMRPortLibrary *portLibrary, char *buf, uintptr_t bufLen, const char *format, int64_t timeMillis) ;
 	/** see @ref omrstr.c::omrstr_ftime_ex "omrstr_ftime_ex"*/
 	uintptr_t (*str_ftime_ex)(struct OMRPortLibrary *portLibrary, char *buf, uintptr_t bufLen, const char *format, int64_t timeMillis, uint32_t flags);
+	/** see @ref omrstr.c::omrstr_current_time_zone "omrstr_current_time_zone"*/
+	int32_t (*str_current_time_zone)(struct OMRPortLibrary *portLibrary, int32_t *hourOffset, uint32_t *minuteOffset, char *buf, uint32_t bufLen) ;
```

Comment:
It would seem much more straight-forward to return a single offset value (minutes or seconds) rather than a signed hour offset and an unsigned minute offset separately, complicating the situation for callers.

Also, `zoneNameBufffer` is a more descriptive name for that parameter; `zoneNameBuffferLen` would go with that.

Reply:
Where it's currently used the offset would be immediately deconstructed into the hour and minute offset anyway. But if the offset itself might be relevant in future I can change it.

Reply:
You might even consider leaving the response in seconds in case higher precision is required at some point.
The value should be negative if east of UTC to match the global convention (e.g. EST is UTC-5 (hours)).

Reply:
I think it's negative for west, since EST is west of the prime meridian. I'll change the returned offset to be a single value.

Reply:
Yes, you're right, the units would be 'seconds west of UTC' which would be negative for zones like EST (which is east of UTC).

Reply:
Is EST being east of UTC a time zone convention I don't know about?

Reply:
Sorry, my correction needs correcting :-( (It's Monday).
EST is in fact _west_ of UTC.